var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -70,8 +70,8 @@ class PyNumpy(PythonPackage):
     depends_on('lapack', when='+lapack')
 
     # Tests require:
-    # TODO: Add a 'test' deptype
-    # depends_on('py-nose@1.0.0:', type='test')
+    # TODO: Add a 'test' deptype, use 'run' deptype for now
+    depends_on('py-nose@1.0.0:', type='run')
```

Comment:
I wouldn't make this change. It adds an unnecessary dependency that isn't used in any way unless tests are required. Although we've been waiting on #1279 for a while now...